src/components/Label/__stories__/label.stories.mdx

```diff
@@ -138,5 +140,18 @@ In case of visual overload, use the secondary label in order to create hirarchy.
   </Story>
 </Canvas>
 
+### Clickable
```

Comment:
Have you talked with the designers about the section this example should be? 
It's arguable, but I think it fits `Variants` better than `Use cases and examples` - like we have in Chips. I'd make sure with one of designers

Reply:
Moved to Variant 👍🏼